server/cli/start.go

```diff
@@ -77,35 +75,30 @@ func runInit(cmd *cobra.Command, args []string) {
 
 // A startCmd command starts nodes by joining the gossip network.
 var startCmd = &cobra.Command{
-	Use: "start --gossip=host1:port1[,host2:port2...] " +
-		"--certs=<cert-dir> " +
-		"--stores=(ssd=<data-dir>,hdd:7200rpm=<data-dir>|mem=<capacity-in-bytes>)[,...]",
-	Short: "start node by joining the gossip network",
+	Use:   "start",
+	Short: "start a node by joining the gossip network",
 	Long: `
-Start Cockroach node by joining the gossip network and exporting key
-ranges stored on physical device(s). The gossip network is joined by
-contacting one or more well-known hosts specified by the --gossip
-flag. Every node should be run with the same list of bootstrap hosts
-to guarantee a connected network. An alternate approach is to use a
-single host for --gossip and round-robin DNS.
-
-Each node exports data from one or more physical devices. These
-devices are specified via the --stores flag. This is a comma-separated
-list of paths to storage directories or for in-memory stores, the
-number of bytes. Although the paths should be specified to correspond
-uniquely to physical devices, this requirement isn't strictly
-enforced.
 
-For example:
+Start a Cockroach node by joining the gossip network and exporting key ranges
+stored on physical device(s). The gossip network is joined by contacting one or
+more well-known hosts specified by the --gossip flag. Every node should be run
+with the same list of bootstrap hosts to guarantee a connected network. An
+alternate approach is to use a single host for --gossip and round-robin DNS.
 
-  cockroach start --gossip=host1:port1,host2:port2 --stores=ssd=/mnt/ssd1,ssd=/mnt/ssd2
+Each node exports data from one or more physical devices. These devices are
+specified via the --stores flag. This is a comma-separated list of paths to
```

Comment:
This sounds like the format is `--stores=/tmp/a,/tmp/b`. We should mention the attributes.